boards/arm/nrf51_vbluno51/doc/nrf51_vbluno51.rst

```diff
@@ -0,0 +1,140 @@
+.. _nrf51_vbluno51:
+
+nRF51-VBLUno51
+##############
+
+Overview
+********
+
+Zephyr uses the nrf51_vbluno51 board configuration to run on the VBLUno51 board.

```

Comment:
I think this should read: 

    Zephyr uses the nrf51_vbluno51 board configuration to run on the VBLUno51 board,
    a VNG Bluetooth Low Energy UNO using an nRF51822 ARM processor.